docs/getting-started/what-is-unlock/features-list.mdx

```diff
@@ -0,0 +1,838 @@
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">QR code tickets and registrations</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Certifications and Credentials</h2>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Onchain certificates & credentials</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Certificate purchase email customization</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Certificate airdrop email customization</div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Certification issuer name support</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Referrer, affiliate, and platform monetization</h2>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Set referrer fees</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Set affiliate fees</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Set platform fees</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Hooks and plugins</h2>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key purchase hook</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key cancel hook</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Valid key hook</div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Token URI hook</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key transfer hook</div>
+        </div>
+      </div>
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key extend hook</div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key grant hook</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Network support</h2>
```

Comment:
Let's remove this section and link to `/core-protocol/unlock/networks`

Reply:
cool. removed.